lib/pwwka/receiver.rb

```diff
@@ -29,13 +29,22 @@ def self.subscribe(handler_klass, queue_name, routing_key: "#.#", block: true)
             receiver.ack(delivery_info.delivery_tag)
             logf "Processed Message on %{queue_name} -> %{payload}, %{routing_key}", queue_name: queue_name, payload: payload, routing_key: delivery_info.routing_key
           rescue => e
+            error_options = {
+               queue_name: queue_name,
+                  payload: payload,
+              routing_key: delivery_info.routing_key,
+                exception: e
+            }
             if Pwwka.configuration.requeue_on_error && !delivery_info.redelivered
-              logf "Retrying an Error Processing Message on %{queue_name} -> %{payload}, %{routing_key}: %{exception}: %{backtrace}", queue_name: queue_name, payload: payload, routing_key: delivery_info.routing_key, exception: e, backtrace: e.backtrace.join(';'), at: :error
+              log_error "Retrying an Error Processing Message", error_options
```

Comment:
Not strictly necessary, but I thought it made the exception-handling logic more clear since the logging statements are quite verbose